spec/deck_spec.rb

```diff
@@ -0,0 +1,30 @@
+require './lib/card'
+require './lib/deck'
+
+RSpec.describe Deck do
+  card_1 = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+  card_2 = Card.new("The Viking spacecraft sent back to Earth photographs and reports about the surface of which planet?", "Mars", :STEM)
+  card_3 = Card.new("Describe in words the exact direction that is 697.5° clockwise from due north?", "North north west", :STEM)
+  deck = Deck.new([card_1, card_2, card_3])
```

Comment:
I would recommend not declaring variables outside of your it blocks. This makes them class variables, which means, as you go about your tests, running methods or passing those certain variables around, it's possible you're changing the attributes associated with each variable. For that reason, I'd define the variables in each it block, OR, look into before(:each) if you're really interested in slimming up your files.